.github/workflows/release.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           FEATURE_FOOTER_BUG_BUTTON: off
           FEATURE_PAYMASTER: on
           MIXPANEL_TOKEN_PUBLIC: ${{ secrets.MIXPANEL_TOKEN_PUBLIC_PROD }}
+          FEATURE_LOYALTY_FLOW: on
```

Comment:
why is it "on"? should be "off", right?

Reply:
yeah, in release.yml I think yes, updated ✅